src/components/CustomerDetails/ManualPassportInput.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, FunctionComponent, useEffect } from "react";
+import { StyleSheet, View } from "react-native";
+import { InputWithLabel } from "../Layout/InputWithLabel";
+import { size } from "../../common/styles";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { DropdownFilterInput } from "../DropdownFilterModal/DropdownFilterInput";
+import { nationalityItems } from "../DropdownFilterModal/nationalityItems";
+import { DropdownItem } from "../DropdownFilterModal/DropdownFilterModal";
+
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center"
+  },
+  inputAndButtonWrapper: {
+    marginTop: size(3),
+    flexDirection: "row",
+    alignItems: "flex-end"
+  },
+  inputWrapper: {
+    flex: 1,
+    marginRight: size(1)
+  }
+});
+
+interface ManualPassportInput {
+  setIdInput: (id: string) => void;
+  submitId: () => void;
+}
+
+export const ManualPassportInput: FunctionComponent<ManualPassportInput> = ({
+  setIdInput,
+  submitId
+}) => {
+  const [selectedCountry, setSelectedCountry] = useState<DropdownItem | null>(
```

Comment:
Noticed this is initialised as nullable, do we need to differentiate between `null` and `undefined` in this case? will `useState<DropdownItem>()` suffice ah?

Reply:
Yup I think `useState<DropdownItem>()` should suffice as I am not really distinguishing between `null` or `undefine` also